bindings/go/src/fdb/range.go

```diff
@@ -90,7 +90,10 @@ type ExactRange interface {
 // that the default zero-value of KeyRange specifies an empty range before all
 // keys in the database.
 type KeyRange struct {
-	Begin, End KeyConvertible
+	// The (inclusive) beginning of the range
+	Begin,
+	// The (exclusive) end of the range
+	End KeyConvertible
```

Comment:
If we are separating these two fields onto separate lines, should we also specify the type on each line? I think stylistically I would prefer that.